src/Jackett.Common/Indexers/ThePirateBay.cs

```diff
@@ -17,9 +17,6 @@
 
 namespace Jackett.Common.Indexers
 {
-    /// <summary>
-    /// The Pirate Bay via API.
-    /// </summary>
```

Comment:
why is the summary removed?

Reply:
This is not meaningful. The class is called ThePirateBay and from my experience if the tracker changes the name or the implementation the comment will be outdated and will be misleading.